scheduler/src/main/java/org/pentaho/platform/scheduler2/quartz/EmbeddedQuartzSystemListener.java

```diff
@@ -112,7 +114,18 @@ public boolean startup( final IPentahoSession session ) {
         if ( logger.isDebugEnabled() ) {
           logger.debug( scheduler.getQuartzScheduler().getSchedulerName() );
         }
-        scheduler.start();
+
+        File schedulerStatusFile = new File(
+          PentahoSystem.getApplicationContext().getSolutionPath( "system" + File.separator + "quartz" ) + File.separator
+            + "scheduler.status" );
+
+        // If the scheduler status file exists and contains "RUNNING" status, start the scheduler
+        if ( schedulerStatusFile.exists() && FileUtils.readFileToString( schedulerStatusFile, StandardCharsets.UTF_8 )
+          .equals( IScheduler.SchedulerStatus.RUNNING.name() ) ) {
+          scheduler.start();
+        } else {
+          scheduler.pause();
+        }
```

Comment:
Could we handle the "scheduler.status" file as a properties file instead? Something like `quartz-state.properties`? The status property would be a property in the file.

We could then use the same API being used to load and save the "normal" `quartz.properties` file: `PentahoSystem.getSystemSettings().getSystemSettingsProperties(.)`.